bedrock/firefox/templates/firefox/accounts.html

```diff
@@ -53,7 +53,6 @@
       <h2 class="c-sub-navigation-title is-summary">
         <a href="{{ url('firefox.accounts') }}" data-link-type="nav" data-link-position="subnav" data-link-name="Firefox Accounts">
           {% if switch('moz-accounts-update') %}
-          <img class="c-sub-navigation-icon" src="{{ static('protocol/img/logos/mozilla/logo.svg') }}" width="24" height="24" alt="">
```

Comment:
Just got confirmation that we won't use the Mozilla logo in the Accounts page after the change launches. Figured I'd just remove it quickly in this PR